PokeAlarm/Cache/Cache.py

```diff
@@ -64,32 +64,36 @@ def update_gym_info(self, gym_id, name, desc, url):
             self._gym_info[gym_id] = {"name": name, "description": desc, "url": url}
 
     def get_egg_expiration(self, gym_id):
-        """ Updates the datetime that the egg expires. """
+        """ Returns the datetime that the egg expires. """
         return self._egg_hist.get(gym_id)
 
     def update_egg_expiration(self, gym_id, expiration):
         """ Updates the datetime that the egg expires. """
         self._egg_hist[gym_id] = expiration
 
     def get_raid_expiration(self, gym_id):
-        """ Updates the datetime that the raid_ expires. """
-        self._raid_hist.get(gym_id)
+        """ Returns the datetime that the raid expires. """
+        return self._raid_hist.get(gym_id)
 
     def update_raid_expiration(self, gym_id, expiration):
-        """ Updates the datetime that the egg expires. """
+        """ Updates the datetime that the raid expires. """
         self._raid_hist[gym_id] = expiration
 
-    def save(self):
-        """ Export the data to a more permanent location. """
+    def cleanup_and_persist(self):
+        """ Invalidates the cache and possibly persists its content. """
         self._clean_hist()
-        log.debug("Cache cleaned!")
+        self._save()
+
+    def _save(self):
+        """ Export the data to a more permanent location. """
+        # Mem cache won't be persisted.
+        pass
 
     def _clean_hist(self):
         """ Clean expired objects to free up memory """
         for dict_ in (self._pokemon_hist, self._pokestop_hist, self._egg_hist, self._raid_hist):
```

Comment:
Change `dict_` to `hist` so I don't have to worry about someone else changing this